llvm/lib/Target/LoongArch/AsmParser/LoongArchAsmParser.cpp

```diff
@@ -1633,6 +1633,9 @@ LoongArchAsmParser::validateTargetOperandClass(MCParsedAsmOperand &AsmOp,
     return Match_Success;
   }
 
```

Comment:
AsmParser and Disassembler changes require a `llvm-mc -filetype=obj ... | llvm-objdump -dr` test in llvm/test/MC/

Reply:
Thanks! These are already covered:

https://github.com/llvm/llvm-project/blob/8d30c73505d3e3f87a4ace7d81ebaa96f5689e7f/llvm/test/MC/LoongArch/Basic/Privilege/valid.s#L24-L26

https://github.com/llvm/llvm-project/blob/8d30c73505d3e3f87a4ace7d81ebaa96f5689e7f/llvm/test/MC/LoongArch/Basic/Privilege/invalid.s#L4-L8